test/e2e/framework/pod/resource.go

```diff
@@ -666,8 +666,8 @@ func GetPodsScheduled(masterNodes sets.String, pods *v1.PodList) (scheduledPods,
 // one provided via the KUBE_TEST_REPO_LIST env variable
 func PatchContainerImages(containers []v1.Container) error {
```

Comment:
Review note: This function PatchContainerImages() has been moved from e2e framework since d45107a00da3d827dd162f70d95d1c4c63e8d95a to  test/e2e/storage/utils/create.go
That is a reason why the file path is different from the original PR(https://github.com/kubernetes/kubernetes/pull/87777)